internal/api/router.go

```diff
@@ -53,5 +53,13 @@ func NewRouter(deps app.Dependencies) *mux.Router {
 	// No version requirement for /ping
 	router.HandleFunc("/ping", pingHandler).Methods(http.MethodGet)
 
+	// organization config
+	router.Handle("/organizationconfig", middleware.JwtAuthMiddleware(getOrganizationConfigHandler(deps.OrganizationConfigService),[]string{constants.UserRole})).Methods(http.MethodGet).Headers(versionHeader, v1)
+
+	router.Handle("/organizationconfig", middleware.JwtAuthMiddleware(createOrganizationConfigHandler(deps.OrganizationConfigService),[]string{constants.UserRole})).Methods(http.MethodPost).Headers(versionHeader, v1)
```

Comment:
same as above, we can remove this and restrict user to create new org.

Reply:
I have commented for now.